packages/compiler/src/myst.ts

```diff
@@ -36,5 +59,95 @@ export function parsePanel(contents: string): Array<MystCard> {
     parsedCard.body = body;
     splitCards.push(parsedCard);
   }
-  return splitCards;
+
+  let myPanel: MystPanel = { cards: splitCards };
+  if (panelStyle) {
+    myPanel = { ...myPanel, style: panelStyle };
+  }
+  return myPanel;
+}
+
+export function parseStyling(contents: string): [string, string] {
```

Comment:
This is great! I think we'll want to reuse this function elsewhere for parsing [directives] in other MyST components. As such, I think `splitDirective` might be a better name here. Also a super-quick comment describing what this method does might be helpful.

[directives]: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html

```suggestion
export function splitDirective(contents: string): [string, string] {
```

Reply:
export function splitDirective(contents: string): [string, string] {